### PR TITLE
Add alias keywords to pipelines

### PR DESCRIPTION
**Pull request by fritze2**
- Combines several triggers into one declaration by making use of `aliasKeywords`.

### CODE REVIEW

File: .ci-orchestrator/pb.yml [hunk 1]
```diff
@@ -347,7 +288,7 @@ steps:
 
 - stepName: Unittest Open Liberty
   workType: RTC
-  projectName: "Liberty Personal Build CI Orchestrator - EBC"
+  projectName: "Open Liberty Personal Build CI Orchestrator - EBC"
```
Comment (websterc87) [inline .ci-orchestrator/pb.yml:291]: Is this going to cause retention issues?

**Review by websterc87 — APPROVED** [review]
Approved but we might need to revisit retention on: Open Liberty Personal Build CI Orchestrator - EBC

File: .ci-orchestrator/pb.yml (hunk 1 above)
Comment (fritze2) [reply .ci-orchestrator/pb.yml:291]: Pruning policy of `Open Liberty Personal Build CI Orchestrator - EBC` is
```
Successful build results to keep: 20
Failed build results to keep: 10
```

versus pruning policy of `Liberty Personal Build CI Orchestrator - EBC` is
```
Successful build results to keep: 30
Failed build results to keep: 15
```

10 fewer successful builds and 5 fewer failed builds. Do you think we should increase that to match if there's disk availability?

Comment (websterc87) [reply .ci-orchestrator/pb.yml:291]: Lets deliver this but monitor over the next few days to see if we are deleting builds to eagerly

**Comment by fritze2** [comment]
#libby

**Pull request merged**